test/askContainer/treeViews/treeViewProviders/helpViewProvider.test.ts

```diff
@@ -0,0 +1,157 @@
+import { HELP_VIEW_ITEMS, EXTERNAL_LINKS } from '../../../../src/constants';
+import { Logger } from '../../../../src/logger';
+
+describe("TreeView_helpViewProvider tests", () => {
+    let helpView: HelpView;
+    let helpViewProvider: HelpViewProvider;
+    let sandbox: sinon.SinonSandbox;
+    let debugSpy: sinon.SinonStub;
+    let verboseSpy: sinon.SinonStub;
+    before(() => {
+        helpView = new HelpView(ext.context);
+        helpViewProvider = new HelpViewProvider(helpView);
+    });
+
+    beforeEach(() => {
+        sandbox = sinon.createSandbox();
+        debugSpy = sandbox.stub(Logger, 'debug');
+        verboseSpy = sandbox.stub(Logger, 'verbose');
+    });
+
+    afterEach(() => {
+        sandbox.restore();
+    });
+
+    describe("getChildren tests", () => {
+        it ('Should add root resources when no element', () => {
+            const expectedGettingStarted = new PluginTreeItem<CustomResource>(
+                HELP_VIEW_ITEMS.GETTING_STARTED, null, vscode.TreeItemCollapsibleState.Collapsed,
+                undefined, undefined, ContextValueTypes.SKILL,
+            );
+            const expectedWhatsNew = new PluginTreeItem<CustomResource>(
+                HELP_VIEW_ITEMS.WHATS_NEW, null, vscode.TreeItemCollapsibleState.None,
+                {
+                    title: 'openUrl',
+                    command: 'ask.container.openUrl',
+                    arguments: [EXTERNAL_LINKS.RELEASE_UPDATES, true, {CommandType: 'RELEASE_UPDATES'}],
+                }, undefined, ContextValueTypes.SKILL,
+            );
+
+            const [gettingStarted, whatsNew, ...others] = helpViewProvider.getChildren();
+
+            assert.ok(debugSpy.calledOnceWith(`Calling method: ${HelpViewProvider.name}.getChildren`));
+            assert.ok(verboseSpy.calledOnceWith(`Calling method: ${HelpViewProvider.name}.addRootResources`));
+            assert.deepStrictEqual(gettingStarted, expectedGettingStarted);
+            assert.deepStrictEqual(whatsNew, expectedWhatsNew);
+            assert.ok(others.length === 0);
+        });
+
+        it ('should add getting started resources when element label is getting started', () => {
+            const fakeElement = new PluginTreeItem<CustomResource>(HELP_VIEW_ITEMS.GETTING_STARTED, null, vscode.TreeItemCollapsibleState.None);
+            const expectedSdkResources = new PluginTreeItem<CustomResource>(
+                HELP_VIEW_ITEMS.GETTING_STARTED_SDK, null,
+                vscode.TreeItemCollapsibleState.Collapsed,
+                undefined, undefined,
+                ContextValueTypes.SKILL,
+            );
+            const expectedAskCli = new PluginTreeItem<CustomResource>(
+                HELP_VIEW_ITEMS.GETTING_STARTED_CLI, null,
+                vscode.TreeItemCollapsibleState.None, {
+                    title: 'openUrl',
+                    command: 'ask.container.openUrl',
```

Comment:
nit: better to use constants for these, and replace them across the tests.

Reply:
this would require to move those string to constant first, will add a todo